lib/gems/pending/appliance_console/cli.rb

```diff
@@ -322,6 +329,23 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def set_server_state
+      service = LinuxAdmin::Service.new("evmserverd")
+      service_running = service.running?
+      case options[:server]
+      when "status"
+        service_running ? "running" : "not running"
```

Comment:
It looks like this doesn't really do anything. I think we can probably just remove the status case.

What do you think @aljesusg ?